frontend/templates/troubleshooting/index.tsx

```diff
@@ -539,11 +532,34 @@ function NavTabs({
 function HreflangUrls({ urls }: { urls: Record<string, string> }) {
    const hreflangs = Object.entries(urls);
    return (
-      <>
+      <Head>
          {hreflangs.map(([lang, url]) => (
             <link rel="alternate" key={lang} hrefLang={lang} href={url} />
          ))}
-      </>
+      </Head>
+   );
+}
+
+function Metadata({
+   metaDescription,
+   title,
+   metaKeywords,
+   canonicalUrl,
+}: {
+   metaDescription: string;
+   title: string;
+   metaKeywords: string;
+   canonicalUrl: string;
+}) {
+   return (
+      <Head>
+         <meta name="description" content={metaDescription} />
+         <meta name="title" content={title} />
+         <meta name="keywords" content={metaKeywords} />
+         <meta name="robots" content="index, follow" />,
+         <link rel="canonical" href={canonicalUrl} />
+         <TagManager />
```

Comment:
Isn't the same issue happening for this component as well?

Reply:
Oh shoot, I didn't notice that.

Yes, you are correct. I will fix it shortly.